packages/database/src/migrations/20210810063133-AddFijiCOVIDDashboardCasesVsDeaths-modifies-data.js

```diff
@@ -0,0 +1,198 @@
+'use strict';
+
+import {
+  insertObject,
+  generateId,
+  findSingleRecord,
+  findSingleRecordBySql,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  code: 'COVID_FJ_Positive_Tests_By_Sub_District',
+  builder: 'analyticArithmetic',
+  config: {
+    formula: 'COVIDTest_FJPosTest',
+    aggregation: {
+      COVIDTest_FJPosTest: {
+        type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+        config: {
+          dataSourceEntityType: 'facility',
+          aggregationEntityType: 'sub_district',
+        },
+      },
+    },
+  },
+};
+
+const generateConfig = () => ({
+  code: `FJ_Covid_Cases_Vs_Deaths`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'sub_district',
+          },
+        },
```

Comment:
Same as Rohan's feedback, it is optional👍 https://github.com/beyondessential/tupaia/pull/2984#discussion_r688164387

Reply:
Thanks... I think you are right but after spending 10 minutes on it I don't think it's worth investing any more time, I'll leave it this time but keep it in mind 🙏.